src/View/Helper/FormHelper.php

```diff
@@ -1024,6 +1024,10 @@ public function input($fieldName, array $options = [])
         $options = $this->_parseOptions($fieldName, $options);
         $options += ['id' => $this->_domId($fieldName)];
 
+        if (strtolower($options['type']) === 'input') {
+                throw new Exception(sprintf('Input is an invalid type option for input field %s', $fieldName));
```

Comment:
The `use Cake\Core\Exception\Exception` statement already existed before my changes. I can change to a `RuntimeException` if you think this is more appropriate?
I also agree that the message could have been quoted better as your example.